modules/csv/csv.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2021 The Gitea Authors. All rights reserved.
```
please also update year in other newly added files